imap_processing/hi/l0/decom_hi.py

```diff
@@ -0,0 +1,21 @@
+from imap_processing import decom, imap_module_directory
+
+
+def decom_packets(packet_file: str, packet_definition_file: str):
```

Comment:
I think a few instruments now use an extremely similar (if not identical) function to this. I wonder if we should make this a function that is generic to all instruments? Maybe put it in the `tools/` folder? This can be outside the scope of this PR, but something to think about.

Reply:
I think this is a great idea @bourque! Would it make sense to make these separate functions for `hi.decom_science(packet_file)`, `hi.decom_housekeeping(packet_file)` where each instrument would have different functions that automatically fill in the XTCE path for the user and just need an input `packet_file` to extract?

Reply:
or should we update `imap_processing/decom.py` and use that for decom and not have function like this?

Reply:
yeah, I think I am going to remove this file. It was not doing much and was doing same thing as `imap_process/decom.py`. And we could have discussion about this at next team meeting or in Slack. I will keep this in mind going forward.